shared/scripts/deploy-template.js

```diff
@@ -7,23 +7,46 @@ const errorOut = message => {
   throw new Error(message)
 }
 
-module.exports = async function deployTemplate(web3, artifacts, templateName, contractName, apps = APPS) {
-  let { ens, owner, verbose, daoFactory, miniMeFactory } = require('yargs')
+module.exports = async function deployTemplate(
+  web3,
+  artifacts,
+  templateName,
+  contractName,
+  apps = APPS
+) {
+  let { ens, owner, verbose, daoFactory, miniMeFactory, register } = require('yargs')
     .option('e', { alias: 'ens', describe: 'ENS address', type: 'string' })
     .option('o', { alias: 'owner', describe: 'Sender address. Will use first address if no one is given.', type: 'string' })
     .option('v', { alias: 'verbose', describe: 'Verbose mode', type: 'boolean', default: false })
     .option('df', { alias: 'dao-factory', describe: 'DAO Factory address. Will deploy new instance if not given.', type: 'string' })
     .option('mf', { alias: 'mini-me-factory', describe: 'MiniMe Factory address. Will deploy new instance if not given.', type: 'string' })
+    .option('r', { alias: 'register', describe: 'Whether the script will register the packages to aragon', type: 'boolean', default: true })
     .help('help')
     .parse()
 
-  if (!web3) errorOut('Missing "web3" object. This script must be run with a "web3" object globally defined, for example through "truffle exec".')
-  if (!artifacts) errorOut('Missing "artifacts" object. This script must be run with an "artifacts" object globally defined, for example through "truffle exec".')
+  if (!web3)
```

Comment:
We should put braces for all of these multiline `if`s; they get very confusing very quickly without.